pyglmnet/pyglmnet.py

```diff
@@ -607,7 +607,8 @@ def fit(self, X, y):
 
             # Update active set
             if self.solver == 'cdfast':
-                ActiveSet[beta[1:] == 0] = 0
+                ActiveSet[beta == 0] = 0
+                ActiveSet[0] = 1.
```

Comment:
what is the shape of `ActiveSet`? n_features + 1?

Reply:
yes. though we never deactivate the intercept because it's not regularized. can be reimplemented with size `n_features`

Reply:
oh yes, as you like. Maybe a comment would do, or reimplementation with `n_features` is also fine. You need to make Travis happy too